assets/messages.yml

```
@@ -108,6 +108,39 @@ messages:
         *Thank you for your report!*
         Could you please include information about your server? Without it we may be unable to reproduce the issue.
 
+        %quick_links%
+      fillname: []
+  attach-video:
+    - project:
+        - mc
+        - mcl
+        - mcpe
+        - mcd
+      name: Attach video (PC)
+      message: |-
+        _We do not have enough information to find the cause of this issue._
+        
+        Please *record a video of this happening* and attach it to this report.
+        If you are on Windows, you can use {{Windows}}+{{Alt}}+{{R}} to open a built-in App for recording game footage.
```

```suggestion
        If you are on Windows, you can use {{Windows}}\+{{Alt}}\+{{R}} to open a built-in App for recording game footage.
```

Jira underlines "Alt" when the `+` characters are not escaped.